### DISCUSSION TITLE
linux/aliyunpan

### DISCUSSION THREAD

**Discussion by giscus** [LaoYutang/blogGiscus#12] [Announcements]
# linux/aliyunpan

阿里云盘自动签到docker部署

https://blog.laoyutang.cn/linux/aliyunpan.html

**Comment by Fantasy9619** [comment]
大佬 这个博客是什么框架

**Reply by LaoYutang** [reply]
vitepress 做的，用了粥里有勺糖大佬的主题包，友情链接里有大佬的博客地址

**Comment by JalonFungar** [comment]
大佬，为什么要自己搭建博客？（这个问题不是调侃，是虚心请教，我在努力游说自己去搭建）
